.mergify.yml

```diff
@@ -137,7 +137,7 @@ pull_request_rules:
     conditions:
       - check-success=apm-ci/pr-merge
       - label=automation
-      - base~=^update-beats
+      - head~=^update-beats
```

Comment:
The condition was wrong, the actual check needs to be performed against the head (current branch) rather than base (target branch).